.circleci/config.yml

```diff
@@ -1,17 +1,17 @@
 orbs:
-  slack: circleci/slack@3.4.1
+  slack: circleci/slack@3.4.2
 version: 2.1
 jobs:
   test:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-2004:202010-01
     steps:
       - checkout
-      - run: docker run --name postgres -p 5432:5432 -d postgres:10.6
+      - run: docker run --name postgres -e POSTGRES_PASSWORD='5ecr3t' -p 5432:5432 -d postgres:latest
```

Comment:
do we need this one?

Reply:
Postgres in general or the password?

Reply:
If the answer is password, then yes. It is mandatory now

Reply:
password, yes

Reply:
also afaik it's better to pin packages versions, so we won't break if there'd be a semver upgrade

Reply:
I've pinned it to 13